.github/workflows/test.yml

```diff
@@ -52,6 +52,7 @@ jobs:
       - name: Run Forge build
         run: |
           forge --version
+          forge install OpenZeppelin/openzeppelin-foundry-upgrades --no-git
```

Comment:
is there a fixed version for the oz contract?

Reply:
It's not just contracts, this installs the plugins that verify storage layout between versions. There are contracts and npm packages.
I'm looking into using just the npm package, to remove the dependency here.